CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing to Affinidi CLI
+
+## Getting started
+
+Clone the repository and run `npm install` command.
+
+Some useful commands:
+
+- Use `npm run dev` to run the CLI locally.
+  > To execute commands, prefix them with `npm run dev` (and ommit the `affinidi` name).
+  > Check out the in-depth instructions in the [CLI ReadMe](README.md) file for using existing commands.
+- Run `npm test` to execute unit tests
+  > You can find them in the `src/test/unit-tests` directory.
+- Run `npm run test:integration` to execute integration tests
+  > You can find them in the `src/test/integration-test` directory.
+- `npm run lint` can be used to check for linting errors
+
+## Implementing a new command
+
+We built the Affinidi CLI with the [Open CLI Framework](https://oclif.io/). We followed their patterns when implementing commands. Please see their [documentation](https://oclif.io/docs/introduction) for a deeper understanding.  
+If you want to implement a new command, you must include an appropriately named file in the `src/commands/` directory. You can also add subcommands by using a nested folder structure.
+
+As an example, let's say you want to add an `auth` command with two subcommands: `get` and `set`.
+
+1. Create `src/commands/auth/` directory
+2. Add your main command logic to `src/commands/auth/index.ts`, and the logic for the subcommands `get` and `set` in their respective files:
+
+```src/
+└── commands/
+    └── auth/
+        ├── index.ts
+        ├── set.ts
+        └── get.ts
+```
+
+For simple commands, you can also take advantage of OCLIF's [command generation tool](https://oclif.io/docs/generator_commands#oclif-generate-command-name) directly in the terminal, which will scaffold a single command file in the `src/commands/` directory and a test file in `src/test/commands/` directory. Please move the created test file to the `src/test/unit-tests/commands/` directory and delete the new `src/test/commands/` directory in order to comply with this repo's test structure.
+
+## Analytics & telemetry
+
+In order to send an analytics event, use the `sendEnabledEvent()` method:
```

Comment:
We only use `sendEnabledEvent()` for sign-in/up and analytics command, but for all the other commands we use `eventsControllerSend()`